### ISSUE TITLE
Silent mode on mobile devices should override the audio switch

### ISSUE THREAD

**Issue by Captain-Quack** [qbreader/website#227]
While testing the new audio features (very cool, by the way), I noticed that the sound didn't turn off when my phone was in silent mode. I'm not sure what the "conventional" action would be for this, but I thought to raise an issue just in case.

**Comment by geoffrey-wu** [comment]
I don't think this would be applicable, since it's a website and should be controlled by your phone sound level.
